src/scanning/ScanningOptionsScreen/ScanningOptionsScreen.jsx

```diff
@@ -0,0 +1,131 @@
+import "./ScanningOptionsScreen.scss";
+import {
+  IonButton,
+  IonContent,
+  IonDatetime,
+  IonDatetimeButton,
+  IonHeader,
+  IonModal,
+  IonPage,
+  IonSearchbar,
+  IonSelect,
+  IonSelectOption,
+  IonTitle,
+  IonToggle,
+  IonToolbar,
+} from "@ionic/react";
+import { useQuery } from "@tanstack/react-query";
+import { useContext, useState } from "react";
+import { searchCandidates } from "../scanning.js";
+import { AppContext } from "../../util/context.js";
+import { Capacitor } from "@capacitor/core";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router";
+
+export const ScanningOptionsScreen = () => {
+  const history = useHistory();
+  const { userInfo } = useContext(AppContext);
+  const [currentCandidateIndex, setCurrentCandidateIndex] = useState(0);
+  const {
+    data: candidates,
+    status,
+    error,
+  } = useQuery({
+    queryKey: ["candidates"],
+    queryFn: () =>
+      searchCandidates({
+        token: userInfo.token,
+        instanceUrl: userInfo.instance.url,
+        platform: Capacitor.getPlatform(),
+      }),
+  });
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+    reset,
+    getValues,
+  } = useForm();
+  const onSubmit = (data) => {
+    console.log(data);
```

Comment:
fine for this PR, but do think about removing console.log() in general, before merging to main. But I'm happy to keep them here until we ship a first version of the app to prod.

Reply:
Alright I can get rid of it. It is not very useful anyway, it was just template code